### PR TITLE
Replace proxy.js by Apache

### PR DESCRIPTION
**Pull request by Nakaner**
Why add logging functionality to proxy.js if we can use Apache?

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -198,4 +257,4 @@ Debian/Ubuntu:
 
  There are a lot of configuration settings in `renderer/config.json`. See https://github.com/rurseekatze/node-tileserver/blob/master/README.md for further details. The default params are ok for most cases. The tileserver has to be restarted after configuration changes.
 
- It is also necassary to change setting parameters in `import/config.cfg`, `api/config.json` and `proxy.js` depending on your envoironment. More information on detailled configuration will be added in future.
+ It is also necassary to change setting parameters in `import/config.cfg` and `api/config.json` depending on your envoironment. More information on detailled configuration will be added in future.
```
Comment (DerDakon) [inline INSTALL.md]: typo "envoironment" (not new, but when you are at it)

File: INSTALL.md [hunk 2]
```diff
@@ -135,6 +138,59 @@ Debian/Ubuntu:
...
+
+ We use Apache. Set up one vhost per subdomain (website, tileserver, API).
+
+ Vhost configuration of the website (`/etc/apache2/sites-available/www.openrailwaymap.org.conf`)
+    <VirtualHost *:80>
+    DocumentRoot /var/www/html/OpenRailwayMap/
+    ServerName www.openrailwaymap.org
+    ServerAlias openrailwaymap.org
+    <Directory /var/www/html/OpenRailwayMap/>
+            Options -Indexes
```
Comment (DerDakon) [inline INSTALL.md]: Please explicitely name the required options (shouldn't be any I guess).

File: INSTALL.md [hunk 3]
```diff
@@ -135,6 +138,59 @@ Debian/Ubuntu:
...
+            AllowOverride AuthConfig FileInfo Limit Indexes Options
```
Comment (DerDakon) [inline INSTALL.md]: use "AllowOverride None" for performance and sanity reasons

File: INSTALL.md [hunk 4]
```diff
@@ -135,6 +138,59 @@ Debian/Ubuntu:
...
+            Order allow,deny
+            allow from all
+            DirectoryIndex index.php index.htm index.html
+            AddType application/x-httpd-php .php
+            AddType application/x-httpd-php-source .phps
```
Comment (DerDakon) [inline INSTALL.md]: This one shouldn't be needed.

**Comment by Nakaner** [comment]
@DerDakon I have fixed the sections (and some more) you have commented.

**Comment by rurseekatze** [comment]
You could also remove the proxy.js script in this pull request.

**Comment by Nakaner** [comment]
@rurseekatze done

**Comment by rurseekatze** [comment]
The formatting of the Apache configs is broken, see https://github.com/Nakaner/OpenRailwayMap/blob/0aab90302e355f4db70f9d6442e4a3a0c35a4c10/INSTALL.md.

The reason could be that there is no newline before the `<VirtualHost *:80>` line. Could you repair it, @Nakaner?

**Comment by Nakaner** [comment]
The markup issue has been fixed.

File: INSTALL.md [hunk 5]
```diff
@@ -135,6 +138,60 @@ Debian/Ubuntu:
...
+    ProxyPreserveHost On
+    ProxyPass / http://localhost:9000/
+    ProxyPassReverse / http://localhost:9000/
+    ErrorLog ${APACHE_LOG_DIR}/tiles.openrailwaymap.org.error.log
+    LogLevel warn
+    CustomLog ${APACHE_LOG_DIR}/tiles.openrailwaymap.org.access.log combined
+    </VirtualHost>
+
+ You will find logfiles of all services at `${APACHE_LOG_DIR}` (e.g. `/var/log/apache2/`). Make sure that your server does not accept request on
+ port 9000 and 9002 from outside (to circumvent the proxy and its logging).
```
Comment (rurseekatze) [inline INSTALL.md]: Could you remove this newline? I think that 188 and 189 are one line.

File: INSTALL.md [hunk 6]
```diff
@@ -173,19 +230,36 @@ Debian/Ubuntu:
...
+    $ sudo a2ensite tiles.openrailwaymap.org.conf
+
+ On other distros this is done by adding following entries to your Apache configuration (vhost do not have to be enabled because your Apache configuration contains `IncludeOptional /etc/apache2/vhosts.d/*.conf`. Check that the proxy modules are enabled. `httpd.conf should contain
+
+    LoadModule mod_proxy
+    LoadModule mod_proxy_http
+
+ Now reload (maybe restart?) Apache. On systems using systemd:
+
+    $ sudo systemctl reload apache2
```
Comment (rurseekatze) [inline INSTALL.md]: On CentOS 7 this is done with `sudo systemctl reload httpd`, on CentOS 6 with `sudo service httpd restart`. Could you add this?

**Comment by Nakaner** [comment]
@rurseekatze I tried to fix your issues. Someone who wants to set up ORM, should know that Apache is sometimes called httpd.

**Pull request merged**